charts/tidb-operator/templates/admission/jobPatch/post-del-secret-job.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and (eq .Values.admissionWebhook.enabled true) (eq .Values.admissionWebhook.certProvider "patch") }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "chart.name" . }}-admission-delete
+  labels:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component:  admission-cert
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded,hook-failed
+spec:
+  template:
+    metadata:
+      name: {{ template "chart.name" . }}-admission-delete
+    spec:
+      restartPolicy: Never
+      serviceAccountName: tidb-admission-webhook-sa
+      containers:
+        - name: "{{.Release.Name}}-job-del"
+          image: bitnami/kubectl:1.15.3
```

Comment:
we should always make image configurable, in case of private or firewalled env

Reply:
fix

Reply:
fix